Index: site.conf
===================================================================
diff --git a/trunk/site.conf b/trunk/site.conf
--- a/trunk/site.conf	(revision 138)
+++ b/trunk/site.conf	(working copy)
@@ -13,7 +13,7 @@
 
 # Location of the music directory, required for editing tags.  This variable
 # is also used to identify the folder for a given file in local cover lookups.
-music_directory: "~/Music/"
+music_directory: "~/music/"
 
 # Only change socket_host if you have multiple network interfaces and want
 # to limit which one it listens on.  "0.0.0.0" listens on all interfaces.
@@ -21,17 +21,17 @@
 
 # Don't run this as root in order to use port 80!  This app was not built
 # to be secure, it must be run with limited access.  Default is port 8080.
-;server.socket_port: 80
+server.socket_port: 8088
 
 # Setting the 'run_as' variable will allow you to start the process as root
 # and drop the privelages to a restricted user account during server startup.
 # This feature is useful when starting client175 during the init process
 # and/or when starting as root to run on port 80.
-;run_as: "user"
+run_as: "jeff"
 
 # Setting environment to production removes the verbose console output
 # and disables auto-reload on file changes.
-;environment: "production"
+environment: "production"
 
 # Setting include_playlist_counts to True will cause the server to count the
 # number of songs and total playtime when listing saved playlists.  If you have
